build/lib/src/builder/post_process_build_step.dart

```diff
@@ -70,26 +72,6 @@ class PostProcessBuildStep {
   Future complete() async {
     await Future.wait(_writeResults.map(Result.release));
   }
-
-  void _addNode(AssetId id) {
-    var node = new GeneratedAssetNode(id,
-        primaryInput: inputId,
-        builderOptionsId: _anchorNode.builderOptionsId,
-        isHidden: true,
-        phaseNumber: _phaseNum,
-        wasOutput: true,
-        isFailure: false,
-        state: GeneratedNodeState.upToDate);
-    _assetGraph.add(node);
-    _anchorNode.outputs.add(id);
-  }
-
-  /// Checks if [id] is a valid output id.
-  void _checkOutput(AssetId id) {
```

Comment:
I see this was moved to the build_runner. Can you explain the move?

Reply:
package:build doesn't know anything about asset graphs or nodes - it only has the general interfaces which can work with any build system. These implementations are build_runner specific.